libs/common-utils/src/environments.ts

```diff
@@ -2,7 +2,7 @@ import { registerOnWindow } from './misc'
 
 const DEFAULT_ENVIRONMENTS_REGEX: Record<EnvironmentName, string> = {
   local: '^(:?localhost:\\d{2,5}|(?:127|192)(?:\\.[0-9]{1,3}){3})',
-  pr: '^(swap-dev-git-[\\w\\d-]+|swap-\\w{9}-)cowswap-dev\\.vercel\\.app',
+  pr: '^(swap-dev-git-[\\w\\d-]+|swap-\\w{9}-)cowswap\\.vercel\\.app',
```

Comment:
Open preview of the current PR, you will see that it makes requests to PROD API instead of Barn. It's happening because of the wrong `isPr` detection.

<img width="1728" alt="image" src="https://github.com/user-attachments/assets/430ea2df-6636-4d84-a17c-9a7aecc5b95c" />

Reply:
Oh, my bad!
I confused widget-configurator with swap-dev 🤦 

You are right, I'll fix it.